packages/service-worker-mock/models/Response.js

```diff
@@ -46,6 +46,23 @@ class Response extends Body {
     });
   }
 
+  /**
+   * Creates a new response with a different URL.
+   * @param url The URL that the new response is to originate from.
+   * @param status [Optional] An optional status code for the response (e.g., 302.)
```

Comment:
It says it's optional, but it's not really. That's okay. I'm fine with making it required, since there is no spec for the default (though it's probably 301).

Reply:
Hi, @zackargyle! Thanks! It seems to be `302` in chrome, I'll update the PR. 

<img width="352" alt="Screen Shot 2019-12-04 at 11 14 11 am" src="https://user-images.githubusercontent.com/1780970/70100860-4c04cb00-1687-11ea-97d4-a7b23ffcbd1b.png">

Reply:
Hey, @zackargyle! I updated the PR with the default value, https://github.com/pinterest/service-workers/pull/128/commits/55e7dc4c6c28a12313f61e5c0e771b8f4fa4b082